cypress/e2e/pages/governance/create/index.spec.ts

```diff
@@ -0,0 +1,532 @@
+      cy.findByTestId("Governance.Create.GettingStarted.Title").should(
+        "have.text",
+        "Getting started"
+      );
+      cy.findByTestId("Governance.Create.GettingStarted.Content").should(
+        "be.visible"
+      );
+      cy.findByTestId("Governance.Create.GettingStarted.Content.GitHub").should(
+        "have.attr",
+        "href",
+        "https://github.com/DeFiCh/dfips/issues"
+      );
+      cy.findByTestId("Governance.Create.GettingStarted.Content.Reddit").should(
+        "have.attr",
+        "href",
+        "https://www.reddit.com/r/defiblockchain/"
+      );
+      cy.findByTestId(
+        "Governance.Create.GettingStarted.Content.ReadHere"
+      ).should(
+        "have.attr",
+        "href",
+        "https://github.com/DeFiCh/dfips/blob/master/README.md"
+      );
+    });
+
+    it("should have step 1", () => {
+      cy.findByTestId("Governance.Create.Step1").should("be.visible");
+
+      cy.findByTestId("Governance.Create.Step1").within(() => {
+        cy.findByTestId("Governance.Create.Step1.Title").should(
+          "have.text",
+          "Step 1: Proposal details"
+        );
+
+        cy.findByTestId("Governance.Create.Step1.Description").should(
+          "have.text",
+          "Enter from GitHub or Reddit the title of the proposal and the type of proposal."
+        );
+      });
+    });
+
+    it("should have step 1 cfp and dfip radio group", () => {
+      cy.findByTestId("Governance.Create.Step1.RadioGroup").should(
+        "be.visible"
+      );
+      cy.findByTestId("Governance.Create.Step1.RadioGroup").within(() => {
+        // CFP radio
+        cy.findByTestId("Governance.Create.Step1.RadioGroup.CFP")
+          .should("be.visible")
+          .within(() => {
+            cy.findByTestId(
+              "Governance.Create.Step1.RadioGroup.CFP.Checked"
+            ).should("be.visible");
+          });
+        // DFIP radio
+        cy.findByTestId("Governance.Create.Step1.RadioGroup.DFIP")
+          .should("be.visible")
+          .within(() => {
+            cy.findByTestId(
+              "Governance.Create.Step1.RadioGroup.DFIP.Unchecked"
+            ).should("be.visible");
+          });
+
+        cy.findByTestId("Governance.Create.Step1.RadioGroup.DFIP").click();
+
+        // DFIP radio
+        cy.findByTestId("Governance.Create.Step1.RadioGroup.DFIP")
+          .should("be.visible")
+          .within(() => {
+            cy.findByTestId(
+              "Governance.Create.Step1.RadioGroup.DFIP.Checked"
+            ).should("be.visible");
+          });
+        // CFP radio
+        cy.findByTestId("Governance.Create.Step1.RadioGroup.CFP")
+          .should("be.visible")
+          .within(() => {
+            cy.findByTestId(
+              "Governance.Create.Step1.RadioGroup.CFP.Unchecked"
+            ).should("be.visible");
+          });
+      });
+    });
+
+    it("should have step 1 textarea inputs for cfp and dfip", () => {
```

Comment:
let's split into individual `it()` clauses for each input

Reply:
Nne scenario for you to consider: if in the future, only `amount requested` field is not found, and it break this test case, you wont know what is missing just from this title `should have step 1 textarea inputs for cfp and dfip`

Reply:
spending extra time now to write a finer title will save you or other future maintainer time